site/server/test/UserStoreSpec.scala

```diff
@@ -0,0 +1,90 @@
+import org.specs2.mutable.Specification
+import org.specs2.specification.BeforeEach
+import test.database.TestDatabase
+import doobie.imports._
+import scalaz.concurrent.Task
+import shared.User
+import com.fortysevendeg.exercises.models.{ UserCreation, UserDoobieStore }
+
+class UserStoreSpec extends Specification with BeforeEach {
+  val db = TestDatabase.create
+  TestDatabase.update(db)
+
+  val transactor: Transactor[Task] = TestDatabase.transactor(db)
+  import transactor.yolo._
+
+  def before = UserDoobieStore.deleteAll.quick.run
+
+  def newUser(
```

Comment:
By the way these params are applied this method may be not necessary since it's delegating all of them in order.